docs/layers.md

```diff
@@ -1317,6 +1318,8 @@ To improve performance, some road segments are merged at low and mid-zooms. To f
 * `description`: OpenStreetMap features
 * `distance`: ski pistes from OpenStreetMap
 * `embankment`: If the road or railway is on an embankment the value will be one of `yes`, `left` or `right` depending on whether the embankment is on both sides, the left side or the right side, respectively.
+* `hgv`: optional property indicating general truck heavy goods vehicle truck access. See below for `kind_detail` list.
+* `hgv_restriction`: optional property indicating limitations to heavy goods vehicle truck access. See below for `kind_detail` list.
```

Comment:
Please also include here (and not below):

- `hgv_restriction_shield_text` - optional property for `hgv_restriction` kind points with values like `5.1m`. Because the units are different per restriction an abbreviation should be provided. Values in meters can be specified with one decimal precision (but value of 5.0m should be given as 5m).

Reply:
Why here? I put it below with the other `point` feature description. Because we have a set of values for the lines and another set for the points. Or would you rather have all defined together? So, merging the section " Road transportation `kind` values (points):" into this one?

Reply:
With your last commits from today now I see what you meant by this comment :)
I thought it was more clear to have it separated, but I am obviously fine to have it together with the additional clarification you added as "this is also for points" and such.
All clear.